.github/workflows/cargo-update.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Test
+
+on:
+  workflow_dispatch:
+  # Run every Monday
+  schedule:
+    - cron: '30 5 * * 1'
+
+jobs:
+  cargo-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: Cargo update
+        run: cargo update
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
```

Comment:
There is a question about whether `committer` and `author` should be specified here...to ensure verified commits. It should be possible to specify the GitHub Actions user - `github-actions@github.com` - to satisfy that requirement...but maybe not worth making that change until the need for it is proven.

`title` may also need to be explicitly specified, if the PRs end up getting titles with the default of `Changes by create-pull-request action`.

Reply:
I don't think setting `committer` and `author` is enough to get verified commits. But I'm not sure it matters anyways. Unlike Dependabot, this will require a human to do the merge. And with a "squash and merge", the commit will be rewritten and (hopefully) signed at that point.

Interesting question about the title... I should say that I pulled most of the options here directly from [this example](https://github.com/peter-evans/create-pull-request/blob/main/docs/examples.md#update-cargo-dependencies).

I'd like to see what happens without specifying those things. We can always update it later.